examples/misc/table_demo.py

```diff
@@ -9,11 +9,11 @@
 import matplotlib.pyplot as plt
 
 
-data = [[  66386,  174296,   75131,  577908,   32015],
```

Comment:
:bike: :derelict_house:  minor preference for leaving this formatting the way it is for human readability.